### ISSUE TITLE
Ctrl+M should toggle Zen Mode not Primary side bar visibility

### ISSUE THREAD

**Issue by radoslav006** [alphabotsec/vscode-eclipse-keybindings#62]
As in title..

**Comment by datMaffin** [comment]
I disagree. While Eclipse hides everything except for the text editor, it does not go into fullscreen. The zen mode on vscode goes into fullscreen hiding everything behind it.

I personally think toggling the vscode sidebar with ctrl-m is the appropriate thing to do.

**Comment by radoslav006** [comment]
`ctrl+m` in eclipse hides left panel, right panel and bottom panel, so hiding only left panel is 1/3 good in terms of clearing your workspace and removing the distractions. Zen mode gets rid of all panels, so it is 3/3 good. Maybe there is third way that could avoid fullscreen, but I don't know if this is possible. I stopped using the vs code some time ago, so I am no longer interested in further investigation.

**Comment by radoslav006** [comment]
Anyone interested in fixing this shortcut?
Maybe there is another solution available, i.e. toggle visibility of the others panels (right & bottom) too?
For me zen mode is the easiest fix, but maybe someone could propose a better solution?

**Comment by p-** [comment]
Hey @radoslav006 

Thanks for your question.

>Maybe there is another solution available, i.e. toggle visibility of the others panels (right & bottom) too?
>For me zen mode is the easiest fix, but maybe someone could propose a better solution?

I'll look into this, but for me Zen mode is not the right solution here.

(I'll leave this issue open as unsolved, but will close the PR)
